### ISSUE TITLE
Keys in active tap groups dosent work behind anticheat

### ISSUE THREAD

**Issue by s0cr4tes** [wasiejen/Free-Snap-Tap#28]
Free-Snap-Tap keys in active tap groups dosent work behind anticheat client.
Works nicely MM, but not custom play like faceit, esportal, esea.

Can you look at it?

**Comment by s0cr4tes** [comment]
[esportal-client-installer_en.zip](https://github.com/user-attachments/files/16727662/esportal-client-installer_en.zip)

This is the client

**Comment by wasiejen** [comment]
Hello @s0cr4tes,

how do key rebinds/replacement and aliases behave? Can you test that out for me?
A simple rebind like ". , d" in the keys file (dot button to d button) and if you move right when `.` pressed.

If that is not working, then they also use a keyboard input listener to directly get the input from the keyboard and ignore anything else.

**Comment by s0cr4tes** [comment]
Nah, its definitely something in the AC that stops it from working in my case. 
The second I close AC and join a listen- or public server MM the program works as intended. 

That's too bad as I do most of my gaming on Faceit/Esportal.

**Comment by wasiejen** [comment]
I have to look further into it, but for that I need more data. If on the AC platform you are using the tap groups are not working, the key groups will also most likely not work. I wanted to you have test it and give me so an additional verification.
With Faceit keyboard events seem to work according to the other open Issue,  just no mouse events - so I need data :-D

**Comment by s0cr4tes** [comment]
Sure, i'll read into your tutorial on rebinds and replacements and try it out later tonight.

**Comment by wasiejen** [comment]
A simple rebind like ". , d" in the keys file (dot/period/point button to d button) and if you move right when . pressed.

Just write ".,d" in the key_group.txt und start the program. now the key `.` (period) should behave like `d`.

**Comment by s0cr4tes** [comment]
I download the files, latest release right. 
I add ".,d" to the empty key_groups.txt and save.
And execute the free_snap_tap.exe.

Do i need to do anymore?
"." does not move me to my right like "D" did before. 

![Untfgtled](https://github.com/user-attachments/assets/a8aa818c-da10-4996-9c1f-7960b36c5da6)

**Comment by s0cr4tes** [comment]
That did the trick . Just added .,d like you said; tried to stafe with (period) - it worked, and then the original WSAD worked aswell.

**Comment by s0cr4tes** [comment]
You did it. Well done :)

**Comment by wasiejen** [comment]
I have done nothing .... xD
Why did it now work???? 
I am confused xD

Are you sure that the snap tap now works? both keys pressed `a` and `d` will not stop you?

**Comment by wasiejen** [comment]
Still waiting for an update? pls? :-D
Want to understand why it now worked. ^^

**Comment by s0cr4tes** [comment]
I think I had to run the .exe as an admin to get it to work with 3rd-party AC.
Normal MM worked without admin-rights.
